tsconfig.json

```
@@ -22,6 +22,7 @@
     "noEmit": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "types": ["node", "jest", "@testing-library/jest-dom"]
+    "types": ["node", "jest", "@testing-library/jest-dom"],
+    "incremental": true
```

```suggestion
    "types": ["node", "jest", "@testing-library/jest-dom"]
```
It doesn't seem to matter: if you pass `--incremental` it works regardless.